### PR TITLE
improve AWS access docs

### PR DESCRIPTION
**Pull request by filiphaftek**
Described who and how can access given AWS account groups.

### CODE REVIEW

File: content/departments/engineering/ship/aws.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ Sourcegraph AWS Organisation is managed by the Ship team. This includes:
...
+| ----------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Management Account (Root)           | Management account is used to managed AWS Identity Center, integrated with OKTA. Terraform access is required to create/delete AWS accounts and assign access for newlt created AWS accounts. | Request Entitle `AWSAdmins` group access and login via [steps](#access-to-an-aws-account-via-web-console)  | Request Entitle `AWSAdmins` group access and follow [steps](#access-to-an-aws-account-via-cliterraform)  |
```
Comment (rafax) [inline content/departments/engineering/ship/aws.md]:
```suggestion
| Management Account (Root)           | Management account is used to manage AWS Identity Center, integrated with OKTA. Terraform access is required to create/delete AWS accounts and assign access for newlt created AWS accounts. | Request Entitle `AWSAdmins` group access and login via [steps](#access-to-an-aws-account-via-web-console)  | Request Entitle `AWSAdmins` group access and follow [steps](#access-to-an-aws-account-via-cliterraform)  |
```

File: content/departments/engineering/ship/aws.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ Sourcegraph AWS Organisation is managed by the Ship team. This includes:
...
+| Cloud production accounts           | Cloud production accounts are customers dedicated accounts for connectivity with customers' code hosts.                                                                                       | Request Entitle `Escalation` group access and login via [steps](#access-to-an-aws-account-via-web-console) | Request Entitle `Escalation` group access and follow [steps](#access-to-an-aws-account-via-cliterraform) |
```
Comment (rafax) [inline content/departments/engineering/ship/aws.md]:
```suggestion
| Cloud production accounts           | Cloud production accounts are customer-dedicated accounts for connectivity with customers' code hosts.                                                                                       | Request Entitle `Escalation` group access and login via [steps](#access-to-an-aws-account-via-web-console) | Request Entitle `Escalation` group access and follow [steps](#access-to-an-aws-account-via-cliterraform) |
```

File: content/departments/engineering/ship/aws.md [hunk 3]
```diff
@@ -7,6 +7,15 @@ Sourcegraph AWS Organisation is managed by the Ship team. This includes:
...
+| Team-dedidcated dev accounts        | Every Team accounts used for non-production purposes. Containing production/customer data is not allowed.                                                                                     | Login via [steps](#access-to-an-aws-account-via-web-console) - no Entitle access required                  | follow [steps](#access-to-an-aws-account-via-cliterraform) - no Entitle access required                  |
```
Comment (rafax) [inline content/departments/engineering/ship/aws.md]:
```suggestion
| Team-dedicated dev accounts        | Team accounts used for non-production purposes. Storing production/customer data is not allowed.                                                                                     | Login via [steps](#access-to-an-aws-account-via-web-console) - no Entitle access required                  | follow [steps](#access-to-an-aws-account-via-cliterraform) - no Entitle access required                  |
```

File: content/departments/engineering/ship/aws.md [hunk 4]
```diff
@@ -7,6 +7,15 @@ Sourcegraph AWS Organisation is managed by the Ship team. This includes:
...
+| Team-dedidcated production accounts | Every Team accounts used for production purposes.                                                                                                                                             | Request Entitle `Escalation` group access and follow [steps](#access-to-an-aws-account-via-web-console)    | Request Entitle `AWSAdmins` group access and follow [steps](#access-to-an-aws-account-via-cliterraform)  |
```
Comment (rafax) [inline content/departments/engineering/ship/aws.md]:
```suggestion
| Team-dedicated production accounts | Every Team accounts used for production purposes.                                                                                                                                             | Request Entitle `Escalation` group access and follow [steps](#access-to-an-aws-account-via-web-console)    | Request Entitle `AWSAdmins` group access and follow [steps](#access-to-an-aws-account-via-cliterraform)  |
```

**Review by rafax — APPROVED** [review]

**Review by daxmc99 — APPROVED** [review]
LGTM

File: content/departments/engineering/ship/aws.md [hunk 5]
```diff
@@ -7,6 +7,15 @@ Sourcegraph AWS Organisation is managed by the Ship team. This includes:
...
+| ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Management Account (Root)          | Management account is used to manage AWS Identity Center, integrated with OKTA. Terraform access is required to create/delete AWS accounts and assign access for newlt created AWS accounts. | Request Entitle `AWSAdmins` group access and login via [steps](#access-to-an-aws-account-via-web-console)  | Request Entitle `AWSAdmins` group access and follow [steps](#access-to-an-aws-account-via-cliterraform)  |
```
Comment (daxmc99) [inline content/departments/engineering/ship/aws.md]:
```suggestion
| Management Account (Root)          | Management account is used to manage AWS Identity Center, integrated with OKTA. Terraform access is required to create/delete AWS accounts and assign access to newly created AWS accounts. | Request Entitle `AWSAdmins` group access and login via [steps](#access-to-an-aws-account-via-web-console)  | Request Entitle `AWSAdmins` group access and follow [steps](#access-to-an-aws-account-via-cliterraform)  |
```

File: content/departments/engineering/ship/aws.md [hunk 6]
```diff
@@ -66,3 +75,13 @@ You can only delete AWS accounts which belong to your team. To delete an AWS acc
 2. Open Pull Request and ask for approval in #discuss-cloud-ops channel.
 
 3. [For Cloud Operations] After merging PR, follow [instructions](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/infrastructure/-/blob/cloud/aws/README.md#create-new-aws-account)
+
+## FAQ
+
+### How can I use AWS IAM Users credentials for automation?
+
+We strongly recommend using AWS SSO role instead of using IAM user credentials for automation. However if you need IAM user credentials for any reason, they are only allowed in non-production accounts.
```
Comment (michaellzc) [inline content/departments/engineering/ship/aws.md:83]: we should say it's mandatory

**Review by michaellzc — APPROVED** [review]

File: content/departments/engineering/ship/aws.md (hunk 6 above)
Comment (filiphaftek) [reply content/departments/engineering/ship/aws.md:83]: For non-production accounts, I think this is still fine to experiment with AWS IAM credentials (most tutorials show it this way, so we do not want to people to waste time learning the alternative ways of access to make quick spike imho), but it's explicitly said that `only allowed in non-production` (so in test accounts).

**Pull request merged**